tests/cra.test.js

```diff
@@ -95,6 +100,50 @@ test('netlify dev cra: robots.txt', async t => {
   t.regex(body, /# https:\/\/www.robotstxt.org\/robotstxt.html/)
 })
 
+
+test('functions rewrite echo without body', async t => {
```

Comment:
New tests to make sure `/api/*` to `/.netlify/functions/:splat` work with framework servers as well.